### DISCUSSION TITLE
Partner with Continuous Delivery Foundation (CDF)

### DISCUSSION THREAD

**Discussion by scottrigby** [open-gitops/project#89] [General]
In today's meeting, @cdavisafc brought up the existing discussion about partnering with the CDF (https://cd.foundation/).

This is a discussion so people can +1/-1, comment, and where a final answer on this can be posted publicly.

**Comment by bricef** [comment]
It would be nice to invite a CDF member to our next public meeting for a quick intro about how they'd like to collaborate.

**Reply by cdavisafc** [reply]
Great idea. I'll drop a note to Tracy Miranda.
